src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.SdkArchiveDiff/VersionIdentifier.cs

```diff
@@ -0,0 +1,257 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+// Copied from https://github.com/dotnet/arcade/blob/903d35c426248ade56719db5bc0947cd7634b297/eng/common/native/init-distro-rid.sh
```

Comment:
Not ideal, but since this is automation that doesn't need to live forever, I'm fine with it.